etc/config/contextBroker

```diff
@@ -57,3 +57,10 @@ BROKER_DATABASE_NAME=orion
 # We need to start the CB with log append mode to not overwrite previous logs and logrotate work correctly
 BROKER_EXTRA_OPS="-multiservice -logAppend"
 
+# The next environment variable generates, if it is defined as 'true', an archive in the 
+# path /var/cb_cores with the core file and the logs when the process crash. When this var is 'true' 
+# the process is started with 'root' user. Use this parameter for debug purposes.
```

Comment:
Another important thing to mention is that the process is run in foreground (-fg) but back-grounded by the shell at the same time with `&`.

Reply:
Correct, this is a difference between the to ways to start the process: with and without core generation

Reply:
Fixed in 6d0e9e9